packages/editor-ui/src/components/RunData.vue

```diff
@@ -513,12 +514,15 @@ export default mixins(
 				}
 				return null;
 			},
-			isMultipleOutputsNodeType(): boolean {
-				return !!this.node && MULTIPLE_OUTPUT_NODE_TYPES.includes(this.node.type);
+			isTriggerNode (): boolean {
+				return !!(this.nodeType && this.nodeType.group.includes('trigger'));
+			},
+			isPinDataNodeType(): boolean {
+				return !!this.node && !PIN_DATA_NODE_TYPES_DENYLIST.includes(this.node.type);
```

Comment:
Nitpick, perhaps a good candidate for the `pinData` mixin - this check is repeated above. Ideally reversed: `hasPinDataAccess(this.node)`

This also avoids having to import/expose `PIN_DATA_NODE_TYPES_DENYLIST`.